src/pages/workspace/accounting/netsuite/import/NetSuiteImportCustomFieldNew/NetSuiteCustomListPicker.tsx

```diff
@@ -0,0 +1,71 @@
+import React, {useState} from 'react';
+import MenuItemWithTopDescription from '@components/MenuItemWithTopDescription';
+import useLocalize from '@hooks/useLocalize';
+import Navigation from '@libs/Navigation/Navigation';
+import type {CustomListSelectorType} from '@pages/workspace/accounting/netsuite/types';
+import CONST from '@src/CONST';
+import type {Policy} from '@src/types/onyx';
+import NetSuiteCustomListSelectorModal from './NetSuiteCustomListSelectorModal';
+
+type NetSuiteCustomListPickerProps = {
+    /** Current value of the selected item */
+    value?: string;
+
+    /** Current connected policy */
+    policy?: Policy;
+
+    /** Callback when the list item is selected */
+    onInputChange?: (value: string, key?: string) => void;
+
+    /** Id of the internalID input to be updated on input change */
+    internalIDInputID?: string;
```

Comment:
```suggestion
    parentInputID?: string;
```

Would that make more sense?

Reply:
Well this is the internal ID of the CustomList and not necessarily parent. Might change the meaning of of this. Any other suggestions?

Reply:
Fair enough. I guess we can keep it as is.